src/main/haskell/kore/src/Kore/Step/Substitution.hs

```diff
@@ -272,7 +272,8 @@ mergeSubstitutionWithPredicate
             return (predicate : predicates, substitution)
 
 normalizePredicatedSubstitution
-    ::  ( MetaOrObject level
+    ::  forall level variable m a .
```

Comment:
Are the explicitly-quantified type variables actually used anywhere else in the scope, besides the initial function declaration?